### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by amirmaroufian**
If the direction of ul or ol element is set to rtl the padding will be padding-right, not padding left. So it's good to mention that.

### CODE REVIEW

File: files/en-us/learn/css/styling_text/styling_lists/index.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ The HTML for our list example looks like so:
 
 If you go to the live example now and investigate the list elements using [browser developer tools](/en-US/docs/Learn/Common_questions/Tools_and_setup/What_are_browser_developer_tools), you'll notice a couple of styling defaults:
 
-- The {{htmlelement("ul")}} and {{htmlelement("ol")}} elements have a top and bottom {{cssxref("margin")}} of `16px` (`1em`) and a {{cssxref("padding-left")}} of `40px` (`2.5em`).
+- The {{htmlelement("ul")}} and {{htmlelement("ol")}} elements have a top and bottom {{cssxref("margin")}} of `16px` (`1em`) and a {{cssxref("padding-left")}} of `40px` (`2.5em`) or {{cssxref("padding-right")}} if the direction is set to rtl.
```
Comment (dipikabh) [inline files/en-us/learn/css/styling_text/styling_lists/index.md:100]:
```suggestion
- The {{htmlelement("ul")}} and {{htmlelement("ol")}} elements have a top and bottom {{cssxref("margin")}} of `16px` (`1em`) and a {{cssxref("padding-left")}} of `40px` (`2.5em`). If the directionality attribute [`dir`](/en-US/docs/Web/HTML/Global_attributes/dir) is set to right-to-left (`rtl`) for `ul` and `ol` elements, in that case {{cssxref("padding-right")}} comes into effect and its default value is `40px` (`2.5em`) .
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for this contribution, @dv3lah.

I have suggested a slight edit to enhance clarity.

**Comment by dv3lah** [comment]
Done, thank you!

**Review by dipikabh — APPROVED** [review]
Thank you, @dv3lah.

**Comment by dipikabh** [comment]
Welcome to the MDN Web Docs project! 🎉

**Comment by dv3lah** [comment]
Thank you!

**Pull request merged**